routers/series.go

```diff
@@ -36,5 +37,8 @@ func SetSeriesRoutes(
 	router.HandleFunc("/v1/series/observations", controllers.GetSeriesObservations(seriesRepository, cacheRepository)).Methods("GET").Queries(
 		"id", "{id:[0-9]+}",
 	)
+	router.HandleFunc("/v1/series/view", controllers.GetSeriesView(seriesRepository, categoryRepository, cacheRepository)).Methods("GET").Queries(
```

Comment:
Something like `package`, `kit`, `box`, `wrapper` etc. would help us avoid the use of `view`, which might be confusing.  If we had a version of this that supported an `ids` GET param, we could support the analyzer view with a request like `/v1/series/kit?ids=150129,150382,153609`.

Reply:
There's actually a nice term from linguistics that fits this application quite exactly, `portmanteau` (which I've been told means something like suitcase in French, but GoogleTrans is not reporting that somehow).... but that might be a little too arcane for our purposes :) Somehow none of the other choices you've suggested "hits" me... and I don't have any better ideas. If you wanna just pick one, I'll move ahead with it. If you want me to pick, I'll probably go with `package`.